### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by AndrewVitrenko**
did drop and merge

### CODE REVIEW

File: Tasks/drop.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-  T.forEach((_) => {
-  { T = [D, X] }
-  if (X.includes(_) && true == 1) {
-delete D[_];
-    { T = T }
+const drop = (dict, ...deleteKeys) => {
+  const dictKeys = Object.keys(dict);
+  for (const key of dictKeys) {
+    if (deleteKeys.includes(key)) {
+      delete dict[key];
```
Comment (tshemsedinov) [inline Tasks/drop.js:8]: Don't change incoming parameters

File: Tasks/merge.js [hunk 2]
```diff
@@ -1,19 +1,7 @@
...
-       );
-        object_3[attribute_name] = object_1[attribute_name]
-  }
-   for (attribute_name in object_2) {
-      (object_3[attribute_name] == object_2[attribute_name])
-     object_3[attribute_name] = object_2[attribute_name]
-    }
-return object_1, object_2, object_3;
-})
+const merge = (obj1, obj2) => ({ ...obj1, ...obj2 });
```
Comment (tshemsedinov) [inline Tasks/merge.js:4]: You can add 1-line version in comments but plz roll back to loops

Comment (AndrewVitrenko) [reply Tasks/merge.js]: fixed

File: Tasks/drop.js (hunk 1 above)
Comment (AndrewVitrenko) [reply Tasks/drop.js]: fixed

**Pull request closed without merging**